openrr-command/src/robot_command.rs

```diff
@@ -481,6 +482,26 @@ pub fn load_command_file_and_filter(file_path: PathBuf) -> Result<Vec<String>, O
         .collect())
 }
 
+fn filter_escape_double_quote(robot_command: &Vec<String>) -> Vec<String> {
```

Comment:
How about using `&[]` instead of `&Vec<>`?

```suggestion
fn filter_escape_double_quote(robot_command: &[String]) -> Vec<String> {
```

And can you add some unit tests for this function?
Or add the documentation code above, which will be tested automatically.

Reply:
Thank you. I updated it and added test for the function.